CONTRIBUTING.md

```diff
@@ -12,12 +12,41 @@ IaSQL is an eventually-consistent design where the SQL tables are the source of
 
 There are a couple of places (like security group IDs) where we can't avoid the ID being generated by the cloud instead of ourselves within postgres. These will need to be nullable columns that we eventually fill in once the security group is created during the creation process. Similarly, things that depend on other things with nullable columns will fail if that column is still null, so `is not null` checks should be included in the queries involving them, though this isn't a hard requirement, as it being null will just cause the API call to fail (but better to not do that when we know it won't work in the first place).
 
-## How to build IaSQL
+## How to run IaSQL locally
 
-This repo houses IaSQL engine which is called by the [cli](https://github.com/iasql/cli). Set your `.env` file based on the values from `src/config.ts`, make sure docker is installed locally and bring up the postgres engine and node.js server by running
+This repo houses the IaSQL engine which is a Node.js HTTP server written in Typescript. Make sure docker is installed locally and bring up the Postgres engine and Node.js server by running
 
 ```
-docker-compose up --build
+IASQL_ENV=local docker-compose up --build
+```
+
+`IASQL_ENV=local` configures the engine to use the values from `src/config/local.ts`. The Postgres superadmin user will be `postgres` and its password `test`. The Node.js server will start on port 8088. To create a new database in your local Postgres engine and connect it to an AWS account (and whatever region you prefer) send the following HTTP request to the local engine:
+
+
+```bash
+curl \
+  --request POST \
+  --url 'http://localhost:8088/v1/db/connect/' \
+  --show-error --silent --fail \
```

Comment:
Minor, but this line of flags is technically not necessary. It will just look a bit uglier without it?

```suggestion
```